Library/Homebrew/formula.rb

```diff
@@ -956,7 +956,7 @@ def bottle_prefix
   # The directory where the formula's installation or test logs will be written.
   # @private
   def logs
-    HOMEBREW_LOGS + name
+    name + HOMEBREW_LOGS
```

Comment:
This is not equivalent.

Reply:
`HOMEBREW_LOGS/name` is probably more appropriate here and won't be changed in this way. Arguably this is a poor use of the `+` operator.